weekly-openmeteo/conftest.py

```diff
@@ -108,3 +113,10 @@ def invalid_openmeteo_data():
             'sunset': ['2024-10-09T18:00', '2024-10-10T18:01', '2024-10-11T18:02',
                        '2024-10-12T18:03', '2024-10-13T18:04', '2024-10-14T18:05',
                        '2024-10-15T18:06']}}]*100
+
+
+@pytest.fixture
+def valid_insertion_data():
+    '''Returns a valid dataset for insertion'''
+    return [[1, '2024-10-09 06:00:00', '2024-10-09 18:00:00'],
+            [2, '2024-10-10 06:01:00', '2024-10-10 18:01:00']]
```

Comment:
An strong, almost exhaustive array of fixtures to be used for mocking test functions.